### DISCUSSION TITLE
Payment availability for russian customers

### DISCUSSION THREAD

**Discussion by Arctomachine** [vercel/community#808] [Help] [answered]
I want to use vercel to host an app for my client. The client is russian corporation (non government), so the payment could be via card or bank transfer. Does vercel accept payments from russian banks? If yes, does it accept bank transfers?

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @Arctomachine! Since we will need to discuss payment information to determine what would work for your situation, please create a new support case at https://vercel.com/help
